src/Program.fs

```diff
@@ -448,6 +451,7 @@ let generate (configFile: string) =
                 // use a low version of FSharp.Core
                 // for better compatibility
                 MSBuildXElement.PackageReferenceUpdate("FSharp.Core", FSharpCoreVersion)
+                MSBuildXElement.PackageReferenceUpdate("FSharp.Control.FusionTasks", FSharpControlFusionTasks)
```

Comment:
Please exclude the Control fusion tasks from here, consumers can add it if they want